## Exercise 1.1

1.	In a print statement, what happens if you leave out one of the parentheses or both?

In [1]:
print "Hello, World"!

SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Hello, World"!)? (<ipython-input-1-cd58c35a9412>, line 1)

ANS: We get a syntax error

2.	If you are trying to print a string, what happens if you leave out one of the quotation marks or both?

In [2]:
print("Hello, World!)

SyntaxError: EOL while scanning string literal (<ipython-input-2-bc0808c61f64>, line 1)

ANS: We get a syntax error

3.	You can use a minus sign to make a negative number like -2. What happens if you put a plus sign before a number? What about 2++2?

In [3]:
2 ++ 2

4

ANS: Here the first ‘+’ represents the normal binary addition operation and the second ‘+’ means integer (+2). So, mathematically 2+(+2) is equal to 4. This is the output.

4.	In math notation, leading zeros are ok, as in 09. What happens if you try this in Python? What about 011?

In [4]:
011

SyntaxError: leading zeros in decimal integer literals are not permitted; use an 0o prefix for octal integers (<ipython-input-4-2e9780ecac58>, line 1)

ANS:	We get a syntax error

5.	What happens if you have two values with no operator between them?

In [5]:
2 4

SyntaxError: invalid syntax (<ipython-input-5-3aca00115f47>, line 1)

ANS: We get a syntax error.

## Exercise 1.2

1. How many seconds are there in 42 minutes 42 seconds?

In [6]:
42 * 60 + 42

2562

ANS: There are 2562 seconds in 42 minutes 42 seconds.

2. How many miles are there in 10 kilometers? Hint: there are 1.61 kilometers in a mile.

In [7]:
10 / 1.61

6.211180124223602

ANS: There are 6.21 miles in 10 kilometers.